01.w_Defects/memory_allocation_failure.c

```diff
@@ -438,7 +438,7 @@ void memory_allocation_failure_012 ()
 	int *ptr[5], a;
 	int flag=10;
 
-    (flag == 10)? (memory_allocation_failure_012_func_002(ptr)) : ( a =20);
+    (flag == 10)? (memory_allocation_failure_012_func_002(ptr)) : (void)( a =20);
```

Comment:
constraint violation: memory_allocation_failure_012_func_002 returns void so the second branch must also have void type.